weblate/lang/tests.py

```diff
@@ -248,6 +248,30 @@ def test_chinese_fuzzy_get(self):
         )
         self.run_create("zh-rCN", "zh_CN", "ltr", "0", "Chinese (zh_CN)", False)
 
+    def test_case_sensitive_fuzzy_get(self):
+        """Test handling of manually created zh-TW, zh-TW and zh_TW languages."""
+        language = Language.objects.create(code="zh_TW", name="Chinese (Taiwan)")
+        language.plural_set.create(
+            number=0,
+            formula="0",
+            source=Plural.SOURCE_DEFAULT,
+        )
+        self.run_create("zh_TW", "zh_TW", "ltr", "0", "Chinese (Taiwan)", False)
+        language = Language.objects.create(code="zh-TW", name="Chinese Taiwan")
```

Comment:
The verbose name will include code here because it's not one of the built-in language definitions:

```suggestion
        language = Language.objects.create(code="zh-TW", name="Chinese Taiwan (zh_TW)")
```